src/components/PortForwarding.vue

```diff
@@ -19,37 +29,72 @@
           @input="handleCheckbox"
         />
       </template>
+      <template #col:listenPort="{row}">
+        <div v-if="serviceBeingEditedIs(row)" class="listen-port-div">
+          <input
+            type="number"
+            :value="serviceBeingEdited.listenPort"
+            class="listen-port-input"
+            v-focus
```

Comment:
I tried adding the two lines 
```
            min="1024"
            max="65535"
```
but they didn't have an effect. Invalid ports are detected by the server, and the error is relayed back to the UI, but I was wondering if this code would prevent making that step. Nope.

Reply:
Yeah, something like what you attempted would only give us simple validation in a form submit event. If we want some more reactive validations, we would have to roll something  ourselves or make use of an existing library like [Vuelidate](https://github.com/vuelidate/vuelidate) or [VeeValidate](https://vee-validate.logaretm.com/v4/)